{{ cookiecutter.repo_name }}/DESCRIPTION

```diff
@@ -2,9 +2,7 @@ Package: {{ cookiecutter.repo_name }}
 Title: {{ cookiecutter.project_name }}
 Version: {{ cookiecutter.project_version }}
 Authors@R:
-    person(given = "{{ cookiecutter.organisation_handle }}",
-           role = c("aut", "cre"),
-           email = "{{ cookiecutter.contact_email }}")
+    person("{{ cookiecutter.organisation_handle }}", , , "{{ cookiecutter.contact_email }}", role = c("aut", "cre"))
```

Comment:
Is there a reason this has changed? Not sure what this means

Reply:
reverted to previous formatting as that is tidier.